dlib/simd/simd8f.h

```diff
@@ -3,11 +3,9 @@
 #ifndef DLIB_sIMD8F_Hh_
 #define DLIB_sIMD8F_Hh_
 
-#include "simd_check.h"
```

Comment:
Same question here.  People might include these files own their own so it would be good if they behaved reasonably in that case.